addons/google_spreadsheet/__manifest__.py

```diff
@@ -25,6 +25,9 @@
         'web.assets_qweb': [
             'google_spreadsheet/static/src/**/*.xml',
         ],
+        'web.qunit_suite_tests': [
+            'google_spreadsheet/static/tests/add_to_google_spreadsheet.js',
```

Comment:
you can simply do this: `google_spreadsheet/static/tests/**/*` instead, to make it easier to modify files